UIKitDZ/ByCodeViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ByCodeViewController.swift
+//  UIKitDZ
+//
+//  Created by muxammed on 17.10.2022.
+//
+
+import UIKit
+
+/// ByCodeViewController экран верстки анкорами кодом
+class ByCodeViewController: UIViewController {
+    
+    let backView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .black
+        return view
+    }()
+    let redView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .red
+        return view
+    }()
+    let yellowView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .yellow
+        return view
+    }()
+    let greenView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .green
+        return view
+    }()
+    
+    func configure() {
+        view.backgroundColor = .white
+        
+        backView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(backView)
+        backView.addSubview(redView)
+        backView.addSubview(yellowView)
+        backView.addSubview(greenView)
+        
```

Comment:
```suggestion
        view.backgroundColor = .white

         backView.translatesAutoresizingMaskIntoConstraints = false
         view.addSubview(backView)
         backView.addSubview(redView)
         backView.addSubview(yellowView)
         backView.addSubview(greenView)
```
можно в отдельный метод